src/custom/ResourceDetailFormatters/useResourceCleanData.ts

```diff
@@ -108,8 +109,13 @@ export const useResourceCleanData = () => {
         links: [
           { nodeName: parsedSpec?.nodeName, label: 'Node' },
           { namespace: resource?.metadata?.namespace, label: 'Namespace' },
-          { serviceAccount: parsedSpec?.serviceAccountName, label: 'ServiceAccount' }
+          {
+            serviceAccount: parsedSpec?.serviceAccountName,
+            label: 'ServiceAccount',
+            resourceCategory: 'Security'
```

Comment:
@amitamrutiya, this behavior should be defined as a modifier of the component's schema/definition and included in the respective model, not centrally embedded. 

We should have precedent for this in terms of ui schema. Is that right, @Yashsharma1911 @aabidsofi19?